src/components/Panel/Panel.js

```diff
@@ -27,7 +27,7 @@ export default observer(({ store }) => {
   const classname = [
     styles.block,
     styles.block__controls,
-    store.annotationStore.viewingAllAnnotations ? styles.hidden : '',
+    store.annotationStore.viewingAll ? styles.hidden : '',
   ].join(' ');
```

Comment:
we should probably filter out empty strings/falsey values from here - but again this is nit-picky

Reply:
```suggestion
  ].filter(Boolean).join(' ');
```